src/__test__/homePage.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import HomePage from '../components/pages/homePage';
+import store from '../redux/store';
+
+test('renders Missions component without errors', () => {
+  render(
+    <MemoryRouter>
+      <Provider store={store}>
+        <HomePage />
+      </Provider>
+    </MemoryRouter>,
+  );
```

Comment:
Well done with the integration tests for your application🤓👍

According to the project requirements, 
![image](https://github.com/Mike47ip/React-Capstone/assets/78251927/e4552e4b-0ed5-4cff-b94c-0b5af5af7d4b)

- [ ] Your app should also have unit tests for several components of the app and some pure functions in Redux code. Please add them🛠🤗

- [ ] The main objective of unit testing is to isolate written code to test and determine if it works as intended which is an important step in the development process. 🛠😊

- [ ] If done correctly, unit tests can detect early flaws in code which may be more difficult to find in later testing stages.

Reply:
Thanks for your detailed review. But one thing I do not understand is why you have an issue with the test when it captures all the 3 components I have in my code?

What else do you expect to see again please?.  I'm lost here. Please clarify or revert